bbb/schemas/payload.yml

```diff
@@ -0,0 +1,77 @@
+id:          http://payload.bbb/v1.schema
```

Comment:
So if using the `Draft4Validator` interface you probably don't have to specify an `id`.
In fact it looks like this package doesn't use the `id`. The JSON schema validators I used for node loads a list of schemas and then relies on `id` when you want to validate against them. This ensures that you can use `$ref` in a schema to reference another schema.

Anyways, not to worry... It seems like you can give it any id, or none at all. IMO you're welcome to publish your schema as `http://schemas.taskcluster.net/buildbot-bridge/v1/payload.json` that way we can embed it in our docs site. You can also publish it anywhere else for embedding in the docs site.

When documenting the bb-bridge it's nice to be able to embed the schema as it documents very clearly what input is supported.. and we can render a nice widget from the schema :)